bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -234,6 +238,12 @@ public String getPrefillService() {
         return prefillService;
     }
 
+    @JsonIgnore
+    @Nullable
+    public String getRoleAttribute() {
```

Comment:
@JsonIgnore should be part of interface.

Reply:
Done!